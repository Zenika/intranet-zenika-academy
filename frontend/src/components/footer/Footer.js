import React from 'react';
import './Footer.scss';
import Linkedin from './linkedin.png';
import Twitter from './twitter.png';
import Youtube from './youtube.png';
import Github from './github.png';

const Footer = () => (
  <article className="content has-text-centered">
    <div className="content has-text-centered">
      <address>
        <strong>ZENIKA ACADEMY - </strong>
        10 rue Milan 75009 Paris France -
        <a href="info@zenika.com"> info@zenika.com - </a>
        <a href="tel:+33(0)1 45 26 19 15">+33(0)1 45 26 19 15</a>
      </address>
      <p className="socialContainer">
        <a href="https://fr.linkedin.com/company/zenika">
          <img className="socialIcon" src={Linkedin} alt="linkedin icon" />
        </a>
        <a href="https://twitter.com/ZenikaIT">
          <img className="socialIcon" src={Twitter} alt="twitter icon" />
        </a>
        <a href="https://www.youtube.com/user/ZenikaITTv">
          <img className="socialIcon" src={Youtube} alt="youtube icon" />
        </a>
        <a href="https://github.com/Zenika">
          <img className="socialIcon" src={Github} alt="github icon" />
        </a>
        <a href="https://blog.zenika.com/"> - ESPACE BLOG - </a>
        <a href="https://www.zenika.com/press">ESPACE PRESSE</a>
      </p>
    </div>
  </article>
);

export default Footer;
