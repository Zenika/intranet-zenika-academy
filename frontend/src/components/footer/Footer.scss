$footer-padding: 2rem 1.5rem 6rem !default;

.footer {
  padding: $footer-padding;
  background-color: #f5f5f5;
}

.socialContainer {
  margin-top: 0.5rem;
}

.socialIcon {
  width: 1rem;
  margin: 0 0.5rem;
}
