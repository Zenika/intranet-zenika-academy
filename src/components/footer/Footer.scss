$footer-padding: 2rem 1.5rem 6rem !default;

.footer {
  padding: $footer-padding;
}

.socialContainer {
  margin-top: 1rem;
}

.socialIcon {
  width: 1.1rem;
  margin: 0 0.5rem;
}
