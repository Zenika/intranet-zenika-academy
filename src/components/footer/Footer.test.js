import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('Footer', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Footer />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('matches snapshot', () => {
    const component = renderer.create(<Footer />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
