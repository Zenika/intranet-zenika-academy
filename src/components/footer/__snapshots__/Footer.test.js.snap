// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<article
  className="content has-text-centered"
>
  <div
    className="content has-text-centered"
  >
    <address>
      <strong>
        ZENIKA ACADEMY - 
      </strong>
      10 rue Milan 75009 Paris France -
      <a
        href="info@zenika.com"
      >
         info@zenika.com - 
      </a>
      <a
        href="tel:+33(0)1 45 26 19 15"
      >
        +33(0)1 45 26 19 15
      </a>
    </address>
    <p
      className="socialContainer"
    >
      <a
        href="https://fr.linkedin.com/company/zenika"
      >
        <img
          alt="linkedin icon"
          className="socialIcon"
          src="linkedin.png"
        />
      </a>
      <a
        href="https://twitter.com/ZenikaIT"
      >
        <img
          alt="twitter icon"
          className="socialIcon"
          src="twitter.png"
        />
      </a>
      <a
        href="https://www.youtube.com/user/ZenikaITTv"
      >
        <img
          alt="youtube icon"
          className="socialIcon"
          src="youtube.png"
        />
      </a>
      <a
        href="https://github.com/Zenika"
      >
        <img
          alt="github icon"
          className="socialIcon"
          src="github.png"
        />
      </a>
      <a
        href="https://blog.zenika.com/"
      >
         - ESPACE BLOG - 
      </a>
      <a
        href="https://www.zenika.com/press"
      >
        ESPACE PRESSE
      </a>
    </p>
  </div>
</article>
`;
